controllers/LikesController.js

```diff
@@ -0,0 +1,166 @@
+import models from '../database/models';
+import ServerResponse from '../modules';
+
+
+const { notFoundError, successResponse } = ServerResponse;
+
+/**
+ * @class likeController
+ * @exports likeController
+ */
+export default class likeController {
+  /**
+   *
+   * @method likePost
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @param {function} next
+   * @returns {object} returns...
+   */
+  static async likeArticle(req, res, next) {
+    try {
+      const { slug } = req.params;
+      const userId = req.user.id;
+
+      const article = await models.Article.findOne({ where: { slug } });
+      if (!article) return notFoundError(req, res);
```

Comment:
Hi could you please change use the `errorResponse` method so a more specific message can be sent back to the client?

Reply:
Ok cool.